rocketpool/node/distribute-minipools.go

```diff
@@ -163,8 +164,11 @@ func (t *distributeMinipools) run(state *state.NetworkState) error {
 	for _, mpd := range minipools {
 		success, err := t.distributeMinipool(mpd, opts)
 		if err != nil {
+			alerting.AlertMinipoolBalanceDistributed(t.cfg, mpd.MinipoolAddress, false)
 			t.log.Println(fmt.Errorf("Could not distribute balance of minipool %s: %w", mpd.MinipoolAddress.Hex(), err))
 			return err
+		} else {
+			alerting.AlertMinipoolBalanceDistributed(t.cfg, mpd.MinipoolAddress, true)
```

Comment:
```suggestion
		success, err := t.distributeMinipool(mpd, opts)
		alerting.AlertMinipoolBalanceDistributed(t.cfg, mpd.MinipoolAddress, err == nil)
		if err != nil {
			t.log.Println(fmt.Errorf("Could not distribute balance of minipool %s: %w", mpd.MinipoolAddress.Hex(), err))
			return err
		} 
```

incredibly minor semantics improvement - easier to see alerting is unconditional at a glance.